remove-user-on-logout/src/main/java/de/intension/resources/admin/VidisAdminRealmResourceProvider.java

```diff
@@ -108,8 +110,7 @@ private List<UserEntity> getListOfUsers(int chunkSize, long lastCreationDate, bo
                 : " ";
         Query userQuery = em.createNativeQuery("select ue.* "
                 + "from user_entity ue "
-                + "where (ue.created_timestamp > :lastTimeStamp "
-                + "or ue.created_timestamp is null) "
+                + "where ue.created_timestamp > :lastTimeStamp "
```

Comment:
so as I understand this is the actual change described by the PR title?
the new config param is additional improvment?

Reply:
No the default tolerance time is also increased. So these are three changes. Don't delete users without created_timestamp. Ignore users younger then 30s (default). If that is to long it is configurable without needing to modify the code.